services/movie.py

```diff
@@ -0,0 +1,37 @@
+from django.db.models import QuerySet
+
+from db.models import Movie, Genre, Actor
+
+
+def get_movies(
+        genres_ids: list[Genre] = None,
+        actors_ids: list[Actor] = None
+) -> QuerySet:

```

Comment:
Indicate a `QuerySet` of what instances you return here and in the other places in the return type annotation